bench/workgen/workgen.h

```diff
@@ -48,9 +48,9 @@ struct OptionsList {
     OptionsList(const OptionsList &other);
     ~OptionsList() = default;
 
-    void add_int(const std::string& name, int default_value, const std::string& desc);
     void add_bool(const std::string& name, bool default_value, const std::string& desc);
     void add_double(const std::string& name, double default_value, const std::string& desc);
+    void add_int(const std::string& name, int default_value, const std::string& desc);
```

Comment:
This is a method/function declaration. So I don't *think* it should be re-ordered but I'm unclear on the exact policy.

Reply:
The ticket does say to reorder the functions

Reply:
@marcbutler

Reply:
Apologies must have missed it.